physics/forkable.hpp

```diff
@@ -84,9 +84,10 @@ class ForkableIterator {
   void CheckNormalizedIfEnd();
 
   // |ancestry_| is never empty.  |current_| is an iterator in the timeline
-  // for |ancestry_.front()|.  |current_| may be at end.
+  // for |ancestry_.back()|.  |current_| may be at end.
   TimelineConstIterator current_;
-  std::deque<not_null<Tr4jectory const*>> ancestry_;  // Pointers not owned.
+  absl::InlinedVector<not_null<Tr4jectory const*>, 2>
```

Comment:
I would think that 3 would work better.  While most trajectories only have 2 elements in this member, most trajectories will go up to 3 (history, psychohistory, prediction) at some point in their lifetime, we might as well reserve space for that case.

Reply:
Done. I added a comment with the reasoning.